test/e2e/framework/sessionconfig.go

```diff
@@ -237,18 +238,29 @@ func (cfg SessionConfig) DeleteFARs() []*ie.IE {
 	}
 }
 
-func (cfg SessionConfig) SessionIEs() []*ie.IE {
+func (cfg SessionConfig) CreateURR (id uint32, triggers uint16) *ie.IE {
+	urr := ie.NewCreateURR(
+		ie.NewURRID(id),
+		ie.NewMeasurementMethod(0, 1, 1), // VOLUM=1 DURAT=1
+		ie.NewReportingTriggers(triggers))
+	// for Quota Validity Time report trigger is 15th bit in host order
+	if (triggers & 0x80) != 0 {
+		urr.Add(ie.NewQuotaValidityTime(time.Now()))
+
+		valTimer, _ := urr.FindByType(ie.QuotaValidityTime)
+		for i, _ := range valTimer.Payload {
+			valTimer.Payload[i] = 0
+		}
+		valTimer.Payload[3] = 10
```

Comment:
As far as I understand, the problem is that Quota Validity Time in go-pfcp version we use wrongly uses `Time` instead of `Duration`? This should probably be noted in a comment above this part of code.
```
// FIXME: go-pfcp QuotaValidityTime definition is incorrect, as it should contain
// a Duration, not Time. Here we set it to be 10 seconds

Reply:
1. It uses Time instead of Duration
2. It overrides provided Time value with some hardcode to make quota never expires. I will add a comment